java/admin/src/main/resources/application.yml

```diff
@@ -46,6 +46,7 @@ spring:
     database: postgresql
     open-in-view: true
     generate-ddl: false
+    show-sql: true
```

Comment:
Can you remove it for the final version?

Reply:
done!